__tests__/intergration.tests.js

```diff
@@ -81,6 +81,23 @@ describe("/api/articles", () => {
         }
       });
   });
+  test("GET:200 should accept optinal query for topics, and if present filter artiles by topic otherwise returnin all articles", ()=>{
```

Comment:
We should be asserting that the articles all have the correct topic property in a for each here